inst/css/BS4/pkgdown.sass

```
@@ -413,12 +407,15 @@ mark {
 }
 
 .algolia-autocomplete .aa-hint {
-  color: #999;
+  color: mix($body-color, $body-bg, 90%);
 }
+
+
+/* "Max" needed until libsass supports https://sass-lang.com/documentation/syntax/special-functions#min-and-max */
 .algolia-autocomplete .aa-dropdown-menu {
-  width: max(100%, 20rem);
-  background-color: #fff;
-  border: 1px solid var(--gray);
+  width: Max(100%, 20rem);
+  background-color: $body-bg;
+  border: 1px solid $table-border-color;
   border-radius: 0.2rem;
```

This way the border radius will respect https://rstudio.github.io/bslib/articles/bs4-variables.html#enable-rounded

```suggestion
  @include border-radius(0.2rem);
```

---

:+1: